README.md

```diff
@@ -83,4 +83,9 @@ docker-compose -p greedybear up
 
 # shut down the application
 docker-compose -p greedybear down
-```
\ No newline at end of file
+```
+Note that GreedyBear *needs* a running instance of ElasticSearch to function.
+If you don't have one, you can make the following changes to make GreeyBear spin up it's own ElasticSearch and Kibana instances:
+
```

Comment:
"...Care! This option would require enough RAM to run the additional containers. Suggested is >=16GB."